climada/engine/test/test_impact_calc.py

```diff
@@ -299,7 +299,7 @@ def test_calc_insured_impact_fail(self):
     def test_minimal_exp_gdf(self):
         """Test obtain minimal exposures gdf"""
         icalc = ImpactCalc(ENT.exposures, ENT.impact_funcs, HAZ)
-        exp_min_gdf = icalc.minimal_exp_gdf('impf_TC')
+        exp_min_gdf = icalc.minimal_exp_gdf('impf_TC', False)
```

Comment:
Maybe make it explicit here as well?
```suggestion
        exp_min_gdf = icalc.minimal_exp_gdf('impf_TC', reassign_centroids=False)
```